packages/go_router/lib/src/path_utils.dart

```diff
@@ -119,41 +119,6 @@ String concatenatePaths(String parentPath, String childPath) {
   return '${parentPath == '/' ? '' : parentPath}/$childPath';
 }
 
-/// Normalizes the location string.
-String canonicalUri(String loc) {
```

Comment:
The logic here is moved to inline in parseRouteInformationWithContext

Reply:
where is `parseRouteInformationWithContext` ? is it inside go_router?

Reply:
yes it is a method in GoRouteInformationParser

Reply:
oh you mean parseRouteInformationWithDependencies